.github/workflows/pr-lint-pr.yml

```diff
@@ -20,7 +17,36 @@ jobs:
     runs-on: ubuntu-latest
     name: Check linked issues
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Checking non-content changes - 🔎
+        shell: pwsh
+        id: check_file_changed
+        run: |
+          # Diff HEAD with the previous commit
+          $diff = git diff --name-only HEAD^ HEAD
+
+          $SourceDiff = $diff | Where-Object { $_ -notmatch '^content/' -and $_ -notmatch '^public/' }
+          $HasDiff = $SourceDiff.Length -gt 0
+
+          echo "hasNonContentChanges=$HasDiff" >> $env:GITHUB_OUTPUT
```

Comment:
recommend avoiding negatives in variable names. Try calling it `hasOnlyContentChanges`

Reply:
I'm using hasCodeChanges which makes more sense in our context.